integ/workflow/wf_state_options_data_attributes_loading/routers.go

```diff
@@ -0,0 +1,292 @@
+package wf_state_options_data_attributes_loading
+
+import (
+	"github.com/gin-gonic/gin"
+	"github.com/indeedeng/iwf/gen/iwfidl"
+	"github.com/indeedeng/iwf/integ/workflow/common"
+	"github.com/indeedeng/iwf/service"
+	"github.com/stretchr/testify/assert"
+	"log"
+	"net/http"
+)
+
+/**
+ * This test has four states
+ *
+ * State1: Creates all Data Attributes keys that will be used in this test
+ * 		- da_wait_until1
+ * 		- da_execute1
+ *		- da_other_key
+ * State2:
+ * 		- Declares State Options containing WaitUntilApiDataAttributesLoadingPolicy
+ * 		- Asserts that (ApiV1WorkflowStateStart) WaitUntil method will load with expected Data Attributes
```

Comment:
also better to clarirfy where is the `Asserts`: in waitUntil or execute

Reply:
I did specify where the Asserts is: `- Asserts that (ApiV1WorkflowStateStart) **WaitUntil** method will load with expected Data Attributes`

But this probably means that it's not very readable. I'll make changes so that this reads better!

Reply:
I made changes to the comment structure so that it is more readable. Let me know what you think, we can also brainstorm about it on a call if you prefer! 👍